IPython/core/profileapp.py

```diff
@@ -264,8 +264,6 @@ def init_config_files(self):
             'IPython.kernel.zmq.kernelapp.IPKernelApp',
             'IPython.terminal.console.app.ZMQTerminalIPythonApp',
             'IPython.qt.console.qtconsoleapp.IPythonQtConsoleApp',
-            'IPython.html.notebookapp.NotebookApp',
-            'IPython.nbconvert.nbconvertapp.NbConvertApp',
```

Comment:
The shim for `IPython.html.notebookapp.NotebookApp` points to the jupyter_notebook version of `NotebookApp` already.